### PR TITLE
[ActivityIndicator] Update accessibility label and value

### PR DESCRIPTION
**Pull request by jmdetloff**
https://www.pivotaltracker.com/story/show/156527550

### CODE REVIEW

**Comment by jverkoey** [comment]
Were you able to auto-generate these changes in some way, or was this a manual change?

File: components/ActivityIndicator/src/MaterialActivityIndicator.bundle/Resources/en.lproj/MaterialActivityIndicator.strings [hunk 1]
```diff
@@ -1,8 +1,11 @@
...
+"MaterialActivityIndicatorAccessibilityLabel" = "Activity Indicator";
 
-/* Accessibility label for indeterminate activity indicator in progress */
-"MaterialActivityIndicatorProgressCompletedAccessibilityLabel" = "%ld Percent Complete";
+/* Accessibility value for indeterminate activity indicator in progress. */
+"MaterialActivityIndicatorInProgressAccessibilityValue" = "In Progress";
 
-/* Accessibility label for activity indicator progress halted */
-"MaterialActivityIndicatorProgressHaltedAccessibilityLabel" = "Progress Halted";
+/* Accessibility value for indeterminate activity indicator in progress. */
```
Comment (jverkoey) [inline components/ActivityIndicator/src/MaterialActivityIndicator.bundle/Resources/en.lproj/MaterialActivityIndicator.strings]: "determinate"?

**Review by jverkoey — APPROVED** [review]
lgtm, but please update the commit description with the method you used to update the strings (whether manual or automated).

File: components/ActivityIndicator/src/MaterialActivityIndicator.bundle/Resources/en.lproj/MaterialActivityIndicator.strings (hunk 1 above)
Comment (jmdetloff) [reply components/ActivityIndicator/src/MaterialActivityIndicator.bundle/Resources/en.lproj/MaterialActivityIndicator.strings]: Yep! Thanks, updated

**Comment by jmdetloff** [comment]
This ended up being a partially manual process. Generate the strings internally, manually create a PR externally.

**Pull request merged**